controller/JobsController.js

```diff
@@ -1,13 +1,31 @@
 var express = require('express');
 var router = express.Router();
+var jobdata = require('../data/jobdata.js')
 
 router.get('/', async (req, res) => {     
   res.render('index') 
 });
 
+router.get('/job-roles', async (req, res) => { 
+  try {
+     var jobs = await jobdata.getJobRoles()
+    console.log(jobs)
+    if (jobs instanceof Error) {
+      res.locals.errormessage = "Failed retrieve jobs: " + jobs.message;
+      res.render('error', req.body)
+    }else if(jobs == null){
+      res.locals.errormessage = "No jobs found";
+      res.render('error', req.body)
+    }
+     else {
+      res.render('jobs.html', { jobRoles: jobs } ) 
+    }
+  } catch(e) {
+    console.log("Failed to load roles: " + e)
+    res.locals.errormessage = "Failed to retrieve jobs: " + e;
```

Comment:
Ah you've got it here too, which highlights it should be in a variable for resuse and then wouldn't be wrong in one place :)